readme.txt

```diff
@@ -289,6 +289,212 @@ Yes, just activate the debug log in the DEBUGGING section of the plugin settings
 * [Changed] DocHooks utility to `micropackage/dochooks`.
 * [Changed] Files utility to `micropackage/filesystem`. Now the plugin has few filesystems which can be accessed easily from outside the plugin.
 * [Changed] View utility to `micropackage/templates`.
+* [Changed] Repeater and recipient field is handled by vue.js.
```

Comment:
Changelog messed because version 6.3.2 and less was archived already. Merge `version/7` back again and make sure the changelog looks good